src/common/menu.rs

```diff
@@ -571,12 +571,66 @@ pub unsafe fn spawn_menu() {
         .unwrap();
 
     let last_url = page_response.get_last_url().unwrap();
-
-    set_menu_from_url(last_url);
+    MENU = set_menu_from_url(MENU, last_url);
 
     let menu_conf_path = "sd:/TrainingModpack/training_modpack_menu.conf";
     std::fs::write(menu_conf_path, last_url).expect("Failed to write menu conf file");
-    unsafe {
-        EVENT_QUEUE.push(Event::menu_open(last_url.to_string()));
+    EVENT_QUEUE.push(Event::menu_open(last_url.to_string()));
+}
+
+pub unsafe fn save_menu_defaults_condition(
```

Comment:
Might this be something that makes more sense to do in-menu rather than in-game? I find that it'll be harder for people to remember this button combination or that it exists, as they don't often read help text past the first time they see a new option.